src/settings/SettingsPage.js

```diff
@@ -63,15 +64,21 @@ export const SettingsPage = ({
         </div>
       </Box> }
 
-    <Box className='mb3 pa4-l pa2'>
-      <div className='lh-copy charcoal'>
+      <Box className='mb3 pa4-l pa2'>
+        <div className='lh-copy charcoal'>
         <Title>{t('app:terms.publicGateway')}</Title>
-        <Trans i18nKey='publicGatewayDescription' t={t}>
-          <p>Choose which <a className='link blue' href="http://docs.ipfs.tech/concepts/ipfs-gateway/#public-gateways" target='_blank' rel='noopener noreferrer'>public gateway</a> you want to use to open your files.</p>
-        </Trans>
-        <PublicGatewayForm/>
-      </div>
-    </Box>
+          <Trans i18nKey='publicSubdomainGatewayDescription' t={t}>
+            <p>Choose which <a className='link blue' href="https://docs.ipfs.tech/concepts/ipfs-gateway/#subdomain" target='_blank' rel='noopener noreferrer'>Subdomain Gateway</a> you want to use when generating shareable links (Default: Subdomain Isolation)</p>
+          </Trans>
+          <PublicSubdomainGatewayForm/>
+        </div>
+        <div className='lh-copy charcoal'>
+          <Trans i18nKey='publicPathGatewayDescription' t={t}>
+            <p>Choose which <a className='link blue' href="https://docs.ipfs.tech/concepts/ipfs-gateway/#path" target='_blank' rel='noopener noreferrer'>Path Gateway</a>  you want to use when generating shareable links (Fallback: path gateway used for CIDs that can't be represented in 63 character DNS label)</p>
```

Comment:
nit: prefer simpler, single sentence. 

mind updating here and in `public/locales/en`?

```suggestion
            <p>Select a fallback <a className='link blue' href='https://docs.ipfs.tech/concepts/ipfs-gateway/#path' target='_blank' rel='noopener noreferrer'>Path Gateway</a> for generating shareable links for CIDs that exceed the 63-character DNS limit.</p>
```